heliopy/data/util.py

```diff
@@ -869,6 +880,37 @@ def _fillval_nan(data, fillval):
 class RemoteFileNotPresentError(RuntimeError):
     pass
 
+def epoch_to_datetime(epoch):
+    """
+    Convert cdf epoch to datetime
+
+    Copied from the cdf2df above to convert non-standard Epoch
+    fields in some cdf files.
```

Comment:
Could you replace the code in `cdf2df` with this function, to avoid duplicating code.